plugins/platform-ui/src/routes.ts

```diff
@@ -1,6 +1,7 @@
 import { AnyComponent } from './index'
 import { parseHash, parsePath, parseQuery } from './location'
 import { Metadata } from '@anticrm/platform'
+import { isEqual } from 'lodash'
```

Comment:
I did try to avoid importing `lodash` in runtime. It's not huge, but extra 30K or so. It's OK to use 'lodash` in our `model` package since `model` is for tooling only and does not used at runtime.

Can we switch to `lodash.isequal` or `lodash-es` as described here: https://stackoverflow.com/questions/43479464/how-to-import-a-single-lodash-function

Reply:
Fixed. Thanks.

Reply:
Hey Andrey @haiodo this is default import, so you should 

```
import isEqual from 'loadash.isEqual'
```

Reply:
Could not make lodash.isequal or lodash-es to work properly with ts-node and jest and in browser at same time, found alternative equals, and it is faster a bit, and small 1.3kb :-) So I assume we will still need some equals method in UI so let's use it instead :-)